content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -238,6 +236,14 @@ Events:
   ...  FailedToRetrieveImagePullSecret  ...  Unable to retrieve some image pull secrets (<regcred>); attempting to pull the image may not succeed.
 ```
 
+## Using images from multiple registries
+
+A pod can have multiple containers, each container image can be from a different registry.
+You can use multiple pull secrets with one pod, and pull secrets can contain multiple credentials.
```

Comment:
```suggestion
You can use multiple `imagePullSecrets` with one pod, and each can contain multiple credentials.
```
given that `imagePullSecrets` is an object field, I think according to the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-object-field-names-and-namespaces), we should code style and camel case that.